tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.psd1

```diff
@@ -62,7 +63,7 @@ PrivateData = @{
         ReleaseNotes = '* the first preview release'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'beta1'
```

Comment:
This is not required. We are following semantic versioning practice. 0.x means preview release.